src/js/dp/get-data.js

```diff
@@ -0,0 +1,17 @@
+import { gtmDataLayerPush } from "../utilities";
+
+document.addEventListener("DOMContentLoaded", function () {
+  const getDataForm = document.forms["get-data-form"];
```

Comment:
I can't find a form called "get-data-form" - is there a paired PR for the dp-frontend-filter-flex-dataset repo that updates this?

Reply:
No PR yet, but will update the `dp-frontend-dataset-controller` with the form name and commit hash

Reply:
cool - so no harm getting it in as long as that is a thing

Reply:
Fails gracefully too as the guard clause is below